client-react/package.json

```diff
@@ -32,9 +32,9 @@
     "uuid": "^8.3.2"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "craco start",
+    "build": "craco build",
+    "test": "craco test",
     "eject": "react-scripts eject"
```

Comment:
Remove eject script as it shouldn't be used

Reply:
removed the eject